protos/certificate.proto

```diff
@@ -31,6 +31,7 @@ message NewCertificatePayload {
     // verify, that the user actually owns the provided certificate.
     string signature_rem = 2;
     string signature_crt = 3;
+    string cert_signer_public_key = 4;
```

Comment:
Please note that this will not be needed when we transfer to node ECDSA keys because public keys will be a part of transaction payload. It is better to mention that in the comments.